lib/helpers.js

```diff
@@ -123,7 +146,9 @@ module.exports = {
    */
   log(msg) {
     /* istanbul ignore next */
-    if (!global.isSilent) console.log(msg)
+    if (global.isSilent) return
+    if (global.isStdout) console.error(msg)
```

Comment:
At this point I would prefer always the --stdout option (hidden to the end user) enabled by default removing completely the use of the `console` api and `process.stderr.write` instead of `console.error`